testsuite/meson.build

```diff
@@ -97,7 +97,8 @@ _testsuite = [
 foreach input : _testsuite
   test(
     input,
-    executable(
+    test_wrapper,
+    args : executable(
```

Comment:
couldn't *_OPTIONS checks be done in meson itself and then just set the env accordingly, possibly with `source`?
This way when calling the binaries outside of meson, the only thing we'd need is:

```
# once..
. scripts/sanitizer-env.sh

call tests how many times we want
```

rather than having to wrap the test call with the test-wrapper.sh for each invocation

Reply:
Ok, let's go with this version that works rather than trying to polish it too much. Thanks

Reply:
we may need to update the README in the testsuite. It used to be the case that we could just `make rootfs && <path/to/test>`, which is not the case anymore. Example:

```
=================================== 16/18 ====================================
test:         test-depmod
start time:   14:11:04
duration:     0.32s
result:       exit status 0
command:      MESON_TEST_ITERATION=1 MALLOC_PERTURB_=117 MSAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1:print_stacktrace=1 LD_LIBRARY_PATH=/home/ldmartin/p/kmod/build/testsuite ASAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1 UBSAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1:print_stacktrace=1 /home/ldmartin/p/kmod/scripts/test-wrapper.sh testsuite/test-depmod
```

Reply:
So... I think this should be handled by the testsuite itself, possibly adding a --disable-sanitizer for all the tests for the case in which "test has been built with sanitizer and we want to run it without". Anyway, something we can think about doing on top.